provider/source/src/segmenter/mod.rs

```diff
@@ -289,44 +285,34 @@ fn generate_rule_break_data(
                         continue;
                     }
 
-                    // The Indic_Conjunct_Break property is separate from the Grapheme_Cluster_Break property.
-                    // See https://unicode.org/reports/tr44/#Indic_Conjunct_Break
-                    if p.name == "InCBConsonant" || p.name == "InCBLinker" || p.name == "InCBExtend"
-                    {
-                        let gcb_extend = gcb_name_to_enum
-                            .get_loose("Extend")
-                            .expect("property name should be valid!");
+                    if p.name == "InCBConsonant" {
+                        for i in 0..(CODEPOINT_TABLE_LEN as u32) {
+                            if let Some(c) = char::from_u32(i) {
+                                if incb.get(c) == IndicConjunctBreak::Consonant {
+                                    properties_map[c as usize] = property_index;
+                                }
+                            }
+                        }
```

Comment:
nit: it's probably more efficient to use `incb.iter_ranges_for_value` than to call `incb.get` a million times

Reply:
The code here does this already, I'm going to fix it in a followup

Reply:
https://github.com/unicode-org/icu4x/pull/6367